packages/themes/doete/tokens/light/brand/brand.json

```diff
@@ -0,0 +1,159 @@
+{
+  "brand": {
+    "color": {
+      "brand": {
+        "emerald": {
+          "$type": "color",
+          "$value": "{primitive.color.base.emerald}"
+        },
+        "gray": {
+          "$type": "color",
+          "$value": "{primitive.color.base.gray}"
+        },
+        "blue": {
+          "$type": "color",
+          "$value": "{primitive.color.base.blue}"
+        },
+        "red": {
+          "$type": "color",
+          "$value": "{primitive.color.base.red}"
+        },
+        "yellow": {
+          "$type": "color",
+          "$value": "{primitive.color.base.yellow}"
+        },
+        "green": {
+          "$type": "color",
+          "$value": "{primitive.color.base.green}"
+        },
+        "purple": {
+          "$type": "color",
+          "$value": "{primitive.color.base.purple}"
+        }
+      },
+      "primary": {
+        "50": { "$type": "color", "$value": "{primitive.color.emerald.50}" },
```

Comment:
Primary color of `doete` is purple.
Can we add a ColorSwatch for purple in the brand.json and use it here instead of the `primitive`?